bridges/bin/millau/runtime/src/lib.rs

```diff
@@ -460,7 +460,7 @@ impl_runtime_apis! {
 
 	impl sp_api::Metadata<Block> for Runtime {
 		fn metadata() -> OpaqueMetadata {
-			Runtime::metadata().into()
+			OpaqueMetadata::new(Runtime::metadata().into())
```

Comment:
those change will need to be applied in https://github.com/paritytech/parity-bridges-common too
cc @HCastano

Reply:
Yep, I can approve here in order to make the checks happy, but for a proper review of the bridges integration please open a PR in `parity-bridges-common`